packages/fixed-decimal/src/FixedDecimal.ts

```diff
@@ -0,0 +1,470 @@
+import type BN from 'bn.js'
```

Comment:
will we be able to get rid of bn.js dep eventually with this?

Reply:
yes